cache/src/lib.rs

```diff
@@ -172,16 +58,18 @@ impl Default for Cache {
 }
 
 /// Copy all info into a plain CacheItem struct.
-/// This is very cheap, and the locking overhead will vastly outstrip making this a non-copy struct.
+/// This is fairly cheap, and the locking overhead will vastly outstrip making this a non-copy struct.
 #[inline]
 fn copy_into_cache_item(cache_item_with_handle: &CacheItem) -> CacheItem {
```

Comment:
Perhaps unrelated to this PR, but I don't think this function buys you much over the `clone` from the default `#[derive(Clone)]` impl, which just performs a clone on each field. For those fields that are `Copy`, their `Clone` impls should be just as efficient.

Reply:
Hmmm, not sure why I felt the need for a custom function. Must have not drank a coffee that day.